pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/converter/ColumnDescriptionsContainer.java

```diff
@@ -0,0 +1,48 @@
+package org.apache.pinot.segment.local.realtime.converter;
+
+import java.util.List;
+
+
+public class ColumnDescriptionsContainer {
```

Comment:
Add javadoc into this class.

Reply:
nit: maybe coming up with a better name like `ColumnIndicesForRealtimeTable` and the like, as it's just for realtime table?

Reply:
it is already under the `realtime` directory. So, it shouldn't require keeping "Realtime" in all class and variable names.J Just makes the code hard to read.